packages/tools/rush-fix-versions/README.md

```diff
@@ -0,0 +1,38 @@
+# @kadena-dev/rush-fix-version
+
+Package allows to align versions to align with `ensureConsistentVersions=true`
+option in rush.json
+
+## Integrate with rush
+
+To use this as a command (e.g. `rush fix-versions`) you can add this as a
+autoinstalled package and add `fix-versions` command to the `command-line.json`.
+
+For reference see
+[maintaining autoinstallers](https://rushjs.io/pages/maintainer/autoinstallers/)
+
+1. initialize the autoinstaller
+   ```sh
+   rush init-autoinstaller --name rush-fix-versions
+   ```
+2. add `@kadena-dev/rush-fix-versions` to the
+   `autoinstallers/rush-fix-versions/package.json`
+3. update the lockfile for the autoinstaller
+   ```sh
+   cd common/autoinstallers/rush-fix-versions
+   rush update-autoinstaller --name rush-fix-versions
+   ```
+4. add the `fix-versions` command to `command-line.json`
+   ```json
+   {
+     "commandKind": "global",
+     "name": "fix-versions",
+     "summary": "Runs fix-versions to get consistent versions across projects",
+     "safeForSimultaneousRushProcesses": false,
+     "autoinstallerName": "rush-fix-versions",
+     // This will invoke ./common/autoinstallers/rush-fix-versions/node_modules/.bin/rush-fix-versions
+     "shellCommand": "rush-fix-versions"
+   },
+   ```
```

Comment:
You already did these steps, right? So this is meant more as a guide to show how did this?

Reply:
Yes, and to show how to use this in other packages. This package will be published, so it could be that others want to use it.